api/src/main/java/module-info.java

```diff
@@ -157,7 +158,24 @@
  * }
  * </pre>
  *
- * <h2>Persistent Field Names</h2>
+ * <h2>Entities</h2>
+ *
+ * <p>Every entity programming model specifies an entity-defining annotation.
+ * For Jakarta Persistence, this is {@code jakarta.persistence.Entity}. For
+ * Jakarta NoSQL, it is {@code jakarta.nosql.Entity}. An entity class is
+ * identified by its entity-defining annotation.</p>
```

Comment:
This wasn't consistent with the spec document, so I adjusted it and copied over the language for unannotated
```suggestion
 * <p>An entity programming model typically specifies an entity-defining
 * annotation that is used to identify entity classes. For Jakarta Persistence,
 * this is {@code jakarta.persistence.Entity}. For Jakarta NoSQL, it is
 * {@code jakarta.nosql.Entity}. A provider may even have no entity-defining
 * annotation and feature a programming model for entity classes where the
 * entity classes are unannotated.</p>
```

Reply:
Yup, in attempting to make it more readable, I changed its meaning. Grr.